imgui_node_editor.h

```diff
@@ -261,7 +261,7 @@ void EndGroupHint();
 ImDrawList* GetNodeBackgroundDrawList(NodeId nodeId);
 
 bool Link(LinkId id, PinId startPinId, PinId endPinId, const ImVec4& color = ImVec4(255, 255, 255, 255), float thickness = 1.0f, bool sameNode = false);
-bool LinkDuplicates(const std::vector<uint64_t>& ids, PinId startPinId, PinId endPinId, const ImVec4& color, float thickness, bool sameNode);
+bool LinkDuplicates(const std::vector<std::pair<uint64_t, ImVec4>>& ids, PinId startPinId, PinId endPinId, const ImVec4& color, float thickness, bool sameNode);
```

Comment:
Isn't this more about supporting colors?

Reply:
yes it is. Maybe the PR title is a little misleading